devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -49,28 +68,62 @@ metadata {
         capability "Configuration"
         capability "Refresh"
         capability "Health Check"
+        capability "Button"
```

Comment:
I don't believe you want to add this to the parent DTH, since I don't think it's ever sending the button events itself

Reply:
Yes, when I added this by referring to[ the DTH](https://raw.githubusercontent.com/krlaframboise/SmartThings/master/devicetypes/krlaframboise/eva-logik-in-wall-smart-switch.src/eva-logik-in-wall-smart-switch.groovy), I couldn't understand the meaning of it at all.Now I removed it